azerothcore/azerothcore-wotlk
# fix(Script/Nexus): Anomalus rift spawning/ empowering timings and Rift Shield cancellation

## CHANGES PROPOSED:
-  Changed the range of charge rifts so it will only charge nearby rifts
- Added smart script to rifts so that when they are destroyed they notify the boss, to cancel the shield.
-  Changed script so that more rifts can be spawned and he shields himself correctly after 51% hp. Will shield once and then spawn a normal rifts, repeat afterwards.
Reference: https://www.youtube.com/watch?v=Flydqb6fuGM


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3274



## TESTS PERFORMED:
- Fight against Anomalus in HC and Normal, checked time between spawns of rifts and rift shields.



## HOW TO TEST THE CHANGES:

1. Set dung to normal or hc
2.  .go c 126599
3. Start boss and check that more than one rift spawns
4. Damage boss to bring him below 51%
5. Check boss now starts to shield himself every so often.

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
